keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -79,66 +82,6 @@ class SqlDeliveryConfigRepository(
         }
     } ?: throw NoDeliveryConfigForApplication(application)
 
```

Comment:
removing this method, because I've re-written it to be in a transaction and to also delete any new bits that were added since this was written